core/src/main/java/org/elasticsearch/Version.java

```diff
@@ -313,6 +313,10 @@ public String toString() {
         return sb.toString();
     }
 
+    public String displayVersion() {
```

Comment:
Ah! I see the static reference now. I think this method is fairly confusing then? Like, it *looks* like it is just about the version that you are calling it on but it does stuff with the *current* build.